packages/indexer/src/handlers/events/evmHost/postRequest.event.handler.ts

```diff
@@ -63,12 +66,13 @@ export async function handlePostRequestEvent(event: PostRequestEventLog): Promis
 		nonce: BigInt(nonce.toString()),
 		source,
 		status: Status.SOURCE,
-		timeoutTimestamp: BigInt(timeoutTimestamp.toString()),
+		timeoutTimestamp: normalizeTimestamp(BigInt(timeoutTimestamp.toString())),
```

Comment:
Don't normalize this, this is directly from the request event, it would break the commitment derivation